server/config.js

```diff
@@ -14,14 +14,14 @@ const feedbackEndpoint = getRequiredEnv(
   '/local-feedback/_doc',
 );
 
-const ALL_PROFILE_FEATURES = [
-  'adjudications',
-  'approvedVisitors',
-  'incentives',
-  'money',
-  'timetable',
-  'visits',
-];
+// const ALL_PROFILE_FEATURES = [
```

Comment:
Does this have to be commented out? I know it's not used now, but commenting it out with no explanation why means it's unclear if this is deprecated or just not used for now.

Reply:
Done.